doc/source/management/data_at_rest_encryption.rst

```diff
@@ -255,25 +255,42 @@ System Variables
   :cli: ``--innodb-temp-tablespace-encrypt``
   :dyn: Yes
   :scope: Global
-  :vartype: Boolean
+  :vartype: Text
   :default: ``Off``
 
-When this option is turned on, server starts to encrypt temporary tablespace
-and temporary |InnoDB| file-per-table tablespaces. The option does not force
-encryption of temporary tables which are currently opened, and it doesn't
-rebuild system temporary tablespace to encrypt data which are already written.
+When this option is set to ``ON``, the server encrypts the temporary
+tablespace. This option does not enforce the encryption of temporary
+tables which are currently open, and it does not rebuild the system
+temporary tablespace to encrypt data which are already written.
 
-Since temporary tablespace is created fresh at each server startup, it will not
-contain unencrypted data if this option specified as server argument.
+The ``ENCRYPTION`` option is not allowed in the ``CREATE TEMPORARY TABLE``
+statement. The ``TABLESPACE`` option cannot be set to `innodb_temporary`. The
+global temporary tablespace datafile ``ibtmp1`` holds temporary table undo logs
+while intrinsic temporary tables and user temporary tables go to the encrypted
+session temporary tablespace.
 
-Turning this option off at runtime makes server to create all subsequent
-temporary file-per-table tablespaces unencrypted, but does not turn off
-encryption of system temporary tablespace.
+Since the global temporary tablespaces (:file:`ibtmp*` data files) and session
+temporary tablespaces (:file:`temp_*.ibt` session temporary tablespace files)
+are created fresh at each server startup, it will not contain unencrypted data
+if this option is specified as a server argument.
+
+Setting this variable to ``OFF`` at runtime makes the server create all
```

Comment:
Setting this variable to OFF and innodb_encrypt_tables=OFF at runtime makes the server create new temporary tables and intrinsic tables to go unencrypted session temporary tablespaces. Existing encrypted user temporary and intrinsic temporary tables remain in encrypted session temporary tablespaces and destroyed only on session disconnect.

Setting this variable to OFF and innodb_encrypt_tables to ON at startup makes only the temporary tablespace datafile ibtmp1 unencrypted but the user created and intrinsic temporary tables go to encrypted session temporary tablespace.

Reply:
Updated & ready for review